torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -2256,7 +2259,7 @@ std::shared_ptr<NCCLComm> ProcessGroupNCCL::getNCCLComm(
       std::make_tuple(pg_uid_, pg_desc_), groupRanks());
 
   RECORD_PARAM_COMMS(
-      0, // seq
+      std::make_tuple(0, false), // seq
```

Comment:
Curious -- is there a corresponding change to `RECORD_PARAM_COMMS` that changes the argument type from int to tuple?

Reply:
Yes. It's here.
I can't get the code position link of this PR, so put screen shot here.
![image](https://github.com/user-attachments/assets/4ec69023-f8f8-469f-b09d-a26c8b3f7226)